src/components/expression/expression.scss

```diff
@@ -44,3 +44,8 @@
     }
   }
 }
+
+.gradient-rect {
+  rx: 10;
```

Comment:
Okay I don't know rects have rx,ry :)  Then please set it via attr to avoid browser compatibilty issue, e.g. rx in style doesn't seem to work in IE11.